bindings/wasm/src/did/wasm_document.rs

```diff
@@ -153,6 +313,39 @@ impl WasmDocument {
     method.revoke_merkle_key(index).wasm_result()
   }
 
+  /// Attaches the relationship to the given method, if the method exists.
+  ///
+  /// Note: The method needs to be in the set of verification methods,
+  /// so it cannot be an embedded one.
+  #[wasm_bindgen(js_name = attachMethodRelationships)]
+  pub fn attach_method_relationships(
+    &mut self,
+    did_url: &WasmDIDUrl,
+    relationship: &MethodRelationshipType,
+  ) -> Result<()> {
+    let relationship: WasmMethodRelationship = relationship.into_serde().wasm_result()?;
```

Comment:
Take `WasmMethodRelationship` as a parameter directly.

Reply:
This will not work, not sure why, but maybe because it's an enum.

Reply:
I tested it in an example and it works just fine.

```rust
#[wasm_bindgen(js_name = attachMethodRelationship)]
pub fn attach_method_relationship(
  &mut self,
  did_url: &WasmDIDUrl,
  relationship: WasmMethodRelationship,
) -> Result<bool> {
  self
    .0
    .attach_method_relationship(&did_url.0, MethodRelationship::from(relationship))
    .wasm_result()
}
```

Reply:
My bad, I was confused by a different error. I fixed it.